.github/workflows/database_backup.yml

```diff
@@ -0,0 +1,28 @@
+name: Backup Database to Azure Storage
+concurrency: build_and_deploy_main
+
+on:
+  workflow_dispatch:
+  # TODO: Uncomment after migration
+  # schedule: # 03:00 UTC
+  #   - cron: "0 3 * * *"
+
+jobs:
+  backup:
+    name: Backup AKS Database (production)
+    if: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Backup postgres
+        uses: DFE-Digital/github-actions/backup-postgres@master
+        with:
+          storage-account: s189p01captdbbkppdsa
+          resource-group: s189p01-capt-pd-rg
+          app-name: claim-additional-payments-for-teaching-production
```

Comment:
Can you double check the exat app name? I think it includes "web" at least

Reply:
Good spot 👀